content/using-packages.md

```diff
@@ -233,15 +233,47 @@ sendTextMessage() {
 }
 ```
 
-<h2 id="atmosphere-overriding">Overriding packages from Atmosphere with a local version</h2>
+<h2 id="overriding-packages">Overriding packages with a local version</h2>
 
-A Meteor app can load packages in one of three ways, and it looks for a matching package name in the following order:
+If you need to modify a package to do something that the published version doesn't do, you can edit a local version of the package on your computer.
+
+<h3 id="npm-overriding">npm</h3>
+
+Let's say you want to modify the `left-pad` npm package. If you haven't already, run inside your app directory:
+
+```bash
+meteor npm install --save left-pad
+```
+
+Now `left-pad` is included in your `package.json`, and the code has been downloaded to `node_modules/left_pad/`. Add the new directory to source control with:
+
+```bash
+git add -f node_modules/left_pad/
+```
+
+Now you can edit the package, commit, and push, and your teammates will get your version of the package. To ensure that your package doesn't get overwritten during an `npm update`, change the default caret version range in your `package.json` to an exact version. Before:
```

Comment:
Alternatively the package could be removed completely from the `package.json` (although that won't help with dependencies)